sygnal/apnstruncate.py

```diff
@@ -38,7 +38,7 @@ def is_too_long(payload: dict, max_length: int = 2048) -> bool:
     return len(json_encode(payload)) > max_length
 
 
-def truncate(payload, max_length=2048):
+def truncate(payload: Dict[Any, Any], max_length: int = 2048) -> Dict[str, Any]:
```

Comment:
```suggestion
def truncate(payload: Dict[str, Any], max_length: int = 2048) -> Dict[str, Any]:
```

If we're returning a `Dict[str, Any]` we should probably restrict the inputs to be the same

Reply:
Thank you :smiley:  I have made the changes